### ISSUE TITLE
Some more issues

### ISSUE THREAD

**Issue by davidhodge931** [neuhausi/canvasXpress#156]
Hi @neuhausi, making great progress!

Some stuff, I noticed checking some examples:
* legend title dropped
* legend title dropped
* strip.text too small (i.e. the labels above facets)
* expand is not working
* where faceting, the legend isn't justified correctly
* the axis line and ticks looks good thickness on gg_bar example, but not gg_histogram
* legend key point size looks too big
* NA is represented as NULL
* histogram representation of data is different and looks incorrect

See below
```
# devtools::install_github('neuhausi/canvasXpress')
library(ggblanket)
library(ggplot2)
library(dplyr)
library(stringr)
library(tidyr)
library(palmerpenguins)
library(patchwork)
library(canvasXpress)

set_blanket(colour = orange)

p <- penguins |>
  gg_point(
    x = flipper_length_mm,
    y = body_mass_g,
    col = island,
    # facet = species,
  )
p
canvasXpress(p, width = 672, height = 500)

p <- penguins |>
  gg_bar(
    x = sex,
    col = island, alpha 
    # facet = species,
  )
p
canvasXpress(p, width = 672, height = 500)

p <- penguins |>
  gg_histogram(
    x = flipper_length_mm,
    col = island,  
    # facet = species,
  )
p
canvasXpress(p, width = 672, height = 500)
```

**Comment by neuhausi** [comment]
Do you mind sending me your email so I can send you the R notebook?

From: David Hodge ***@***.***>
Date: Thursday, April 11, 2024 at 4:37 PM
To: neuhausi/canvasXpress ***@***.***>
Cc: Isaac Neuhaus ***@***.***>, Mention ***@***.***>
Subject: [neuhausi/canvasXpress] Some more issues (Issue #156)

Hi @neuhausi<https://github.com/neuhausi>, making great progress!

Some stuff, I noticed checking some examples:

  *   legend title dropped
  *   legend title dropped
  *   strip.text too small (i.e. the labels above facets)
  *   expand is not working
  *   where faceting, the legend isn't justified correctly
  *   the axis line and ticks looks good thickness on gg_bar example, but not gg_histogram
  *   legend key point size looks too big

See below

# devtools::install_github('neuhausi/canvasXpress')

library(ggblanket)

library(ggplot2)

library(dplyr)

library(stringr)

library(tidyr)

library(palmerpenguins)

library(patchwork)

library(canvasXpress)



set_blanket(colour = orange)



p <- penguins |>

  gg_point(

    x = flipper_length_mm,

    y = body_mass_g,

    col = island,

    # facet = species,

  )

p

canvasXpress(p, width = 672, height = 500)



p <- penguins |>

  gg_bar(

    x = sex,

    col = island, alpha

    # facet = species,

  )

p

canvasXpress(p, width = 672, height = 500)



p <- penguins |>

  gg_histogram(

    x = flipper_length_mm,

    col = island,

    # facet = species,

  )

p

canvasXpress(p, width = 672, height = 500)

**Comment by neuhausi** [comment]
Just released version 49.1. Please check and close all issues as a appropriate.
Isaac

**Comment by davidhodge931** [comment]
I don't have any time to work further on this, sorry @neuhausi 

I'm trying to finish my own project, and then have a break from open source for a while.

Feel free to close any issues I've raised :)
